Crane/buildscripts/toml-updater-config.gradle

```diff
@@ -0,0 +1,30 @@
+versionCatalogUpdate {
```

Comment:
Add license header to this file? Same comment for all introduced files. https://github.com/google/addlicense can help here